File: backend_springboot/src/main/java/com/zinemaapp/zinemaapp/domain/Film.java
package com.zinemaapp.zinemaapp.domain;

import java.time.LocalDate;
import java.util.List;

public class Film {
    private int id;
    private String title;
    private String originalTitle;
    private LocalDate releaseDate;
    private String synopsis;
    private String poster;
    private double rating;
    private List<Person> actors;
    private Person director;
    private List<Genre> genres;
}

File: app_kotlin/app/src/main/java/com/example/proyecto_gestion_peliculas/domain/Film.kt
package com.example.proyecto_gestion_peliculas.domain

import java.time.LocalDate

data class Film(
    val id : Int,
    val title : String,
    val releaseDate : LocalDate?,
    val genres: List<Genre>,
    val director : String?,
    val actors : List<Actor>,
    val synopsis : String,
    val poster : String?,
    val rating: Double
)
